[en]update insight-alert docs (#5108)
DaoCloud/DaoCloud-docs: docs/en/docs/insight/user-guide/alert-center/inhibition.md

immediate attention. The purpose of this feature is to reduce unnecessary Alert information that may disturb 
operations personnel, allowing them to focus on more critical issues.

> Alarm inhibition recognizes and ignores certain alarms by defining a set of rules to deal with specific conditions. 

Reviewer: ```suggestion
Alert inhibition recognizes and ignores certain alerts by defining a set of rules to deal with specific conditions. 
```
alarm -> alert

There are mainly the following conditions:

> - Parent-child inhibition: when a parent alarm (e.g., a crash on a node) is triggered, all child alarms aroused by 

Reviewer: e.g. 要少用，参阅：
https://kubernetes.io/docs/contribute/style/style-guide/#avoid-latin-phrases

2. Click **Create Inhibition**, and set the name and rules for the inhibition.

    - Source Alert: Alerts that match the specified label conditions will be compared with those that meet the 

>       suppression conditions. Alerts that do not meet the suppression conditions will be sent to the user as usual.

Reviewer: ```suggestion
      inhibition conditions. Alerts that do not meet the inhibition conditions will be sent to the user as usual.
```
用词要跟 UI 一致，UI 上是 inhibition

2. Click **Create Inhibition**, and set the name and rules for the inhibition.

    !!! note

        The problem of avoiding multiple similar or related alerts that may be triggered by the same issue is achieved 

>         by defining a set of rules to identify and ignore certain alerts through [Rule Details](#_3) and [Alert Details](#_4).

Reviewer: 锚点要改成英文的

    | Description | The description can contain any characters and can be up to 256 characters long. |
    | Cluster | The cluster where the inhibition rule applies. |
    | Namespace | The namespace where the inhibition rule applies. |
    | Source Alert | Specifies the matching conditions for the source Alert (the Alert that triggers the inhibition).<br /><br />Value range explanation:<br /> - **Alert Level**: The level of metric or event Alerts, can be set as: Critical, Major, Minor.<br /> - **Resource Type**: The resource type corresponding to the Alert object, can be set as: Cluster, Node, StatefulSet, Deployment, DaemonSet, Pod. <br /> - **Labels**: Alert identification attributes, consisting of label name and label value, supports user-defined values. |

>     | Inhibition | Specifies the matching conditions for the target Alert (the Alert to be suppressed). |

Reviewer: ```suggestion
    | Inhibition | Specifies the matching conditions for the target Alert (the Alert to be inhibited). |
```

    | Inhibition | Specifies the matching conditions for the target Alert (the Alert to be inhibited). Alarms that meet all the conditions will no longer be sent to the user. |
    | Equal | Specifies the list of labels to compare to determine if the source Alert and target Alert match. Inhibition is triggered only when the values of the labels specified in `equal` are exactly the same in the source and target Alerts. The `equal` field is optional. If the `equal` field is omitted, all labels are used for matching. |

3. Click **OK** to complete the creation and return to Inhibition list. Click the inhibition rule name to view the rule details.

### View Rule Details

> In the left navigation bar, select **Alert** -> **Alert Policy**, and click the policy name to view the rule details.

Reviewer: ```suggestion
In the left navigation bar, select **Alert** -> **Alert Policy**, and click the policy name to view the policy details.
```
policy 是一种 rule，所以描述中的 rule 可能都要改成 policy
